src/daft-plan/src/builder.rs

```diff
@@ -0,0 +1,29 @@
+use std::sync::Arc;
+
+use daft_core::schema::SchemaRef;
+
+use crate::logical_plan::LogicalPlan;
+use crate::ops;
+use crate::source_info;
+
+#[derive(Clone)]
+pub struct LogicalPlanBuilder {
+    _plan: Arc<LogicalPlan>,
```

Comment:
Probably don't need the underscore prefix to denote that the field is private, since struct fields are private by default and would only be made public with the `pub plan` qualifier.